### DISCUSSION TITLE
Note Title Formatting Options

### DISCUSSION THREAD

**Discussion by overwatcheddude** [streetwriters/notesnook#5236] [Ideas]
Currently, users cannot customize the formatting of the note title. I would like to suggest allowing users to apply styles like bold,  underline and strike-through to the title.

![image](https://github.com/streetwriters/notesnook/assets/37483931/b6b0b0de-2dbc-4b19-991c-5658d12f9c26)

**Comment by fallsdevil** [comment]
Yes! I was trying to customize it by wanting to replace the "-" with a dot and leave the year with just 2 digits and I don't know how to do it...
